src/data_provider/src/osinfo/sysOsParsers.cpp

```diff
@@ -137,15 +137,30 @@ static bool findVersionInStream(std::istream& in,
 bool UnixOsParser::parseFile(std::istream& in, nlohmann::json& info)
 {
     constexpr auto SEPARATOR{'='};
-    static const std::map<std::string, std::string> KEY_MAP
+    bool ret = false;
+
     {
```

Comment:
Question: why this grouping?

Reply:
This local scope is defined here in order to allow a redefinition of local variable name "KEY_MAP" in the following lines (also defined as a local scope).

Reply:
Ok, I see below, but maybe it seems excessive, consider just another variable name

Reply:
Ok, I change it